__docs__/wonder-blocks-card/card.argtypes.tsx

```diff
@@ -0,0 +1,131 @@
+import type {ArgTypes} from "@storybook/react-vite";
+
+export default {
+    children: {
+        control: {type: "text"},
+        table: {
+            type: {summary: "React.ReactNode"},
+        },
+        type: {name: "other", value: "React.ReactNode", required: true},
+    },
+    backgroundColorStyle: {
```

Comment:
`backgroundColorStyle` and `borderRadiusStyle` prop names in the argtypes can be updated to the new names!